res_syntax/compiler-libs-406/oprint.ml

```diff
@@ -1,773 +0,0 @@
-(**************************************************************************)
```

Comment:
This outcome printer probably has a small changes that was not yet present in the compilers's outcome printer and should be brought over.